src/com/facebook/buck/swift/SwiftRuntimeNativeLinkable.java

```diff
@@ -80,7 +80,11 @@ public NativeLinkableInput getNativeLinkableInput(
               "-Xlinker",
               "-rpath",
               "-Xlinker",
-              "@executable_path/Frameworks"));
+              "@executable_path/Frameworks",
+              "-Xlinker",
+              "-rpath",
+              "-Xlinker",
+              "@loader_path/Frameworks"));
```

Comment:
During testing, ios loads the bundle and inject the test code to the process, therefore we need to add loader path to the list.
You might ask, why am I always adding this, even when it is not for test? Well, xcode is doing the same thing and it doesn't hurt by having it in.